<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setbox</class>
 <widget class="QMainWindow" name="Setbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>基本参数</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>显示</string>
          </property>
          <widget class="QLabel" name="ImgSize_text">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>缩放</string>
           </property>
          </widget>
          <widget class="QSlider" name="ImgSize_control">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>20</y>
             <width>191</width>
             <height>22</height>
            </rect>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="pageStep">
            <number>0</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="sliderPosition">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QLabel" name="ImgSize_text_percent">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>20</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>基本参数</string>
          </property>
          <widget class="QLineEdit" name="Name_show">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>20</y>
             <width>241</width>
             <height>20</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="Name_text">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>21</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>名称</string>
           </property>
          </widget>
          <widget class="QLabel" name="Introduction_text">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>介绍</string>
           </property>
          </widget>
          <widget class="QPlainTextEdit" name="Introduction_show">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>50</y>
             <width>241</width>
             <height>91</height>
            </rect>
           </property>
           <property name="plainText">
            <string/>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ImgSize_control</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Setbox</receiver>
   <slot>ImgSize_control_valueChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ImgSize_control_valueChange()</slot>
 </slots>
</ui>
